src/components/SolarFlareTypeForm.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { ReactTypeformEmbed } from 'react-typeform-embed'
+
+const URL = process.env.GATSBY_TYPEFORM_SOLAR_FLARE_URL
+
+type SolarFlareTypeFormProps = {
+  disclosure: JSX.Element
+  popup: boolean
+}
+
+const SolarFlareTypeForm = ({
+  disclosure,
+  popup = true
+}: SolarFlareTypeFormProps): JSX.Element => {
+  const typeformRef = React.useRef()
+
+  const openForm = React.useCallback(() => {
+    typeformRef.current?.typeform?.open()
```

Comment:
Getting into the `?.` of things 💪

Reply:
yeah, much nicer than writing it all out!